clang/test/CodeGenHLSL/builtins/StructuredBuffers-constructors.hlsl

```diff
@@ -5,17 +5,27 @@
 
 StructuredBuffer<float> Buf : register(t10);
 RWStructuredBuffer<float> Buf2 : register(u5, space1);
+AppendStructuredBuffer<float> Buf3 : register(u3);
+ConsumeStructuredBuffer<float> Buf4 : register(u4);
 
-// CHECK: %"class.hlsl::StructuredBuffer" = type { target("dx.RawBuffer", float, 0, 0), float }
-// CHECK: %"class.hlsl::RWStructuredBuffer" = type { target("dx.RawBuffer", float, 1, 0), float }
+// CHECK: %"class.hlsl::StructuredBuffer" = type { target("dx.RawBuffer", float, 0, 0)
```

Comment:
It doesn't seem like you changed anything about StructuredBuffer / RWStructuredBuffer, do you know why the trailing "float }" was removed from the test? Is it because it's redundant and not useful for the test?

Reply:
The trailing float is a field temporarily added to resource buffer declarations and it is used in a placeholder implementation of the subscript operators. It will go away once llvm/llvm-project#95956 is implemented. I am removing it from the baseline to minimize the test changes when that happens.